packages/regl-worldview/src/commands/FilledPolygons.js

```diff
@@ -50,18 +50,17 @@ type Props = {
 export default function FilledPolygons({ children: polygons = [], getHitmapId }: Props) {
   const triangles = [];
   for (const poly of polygons) {
-    const { points } = poly;
+    const { points, id, color } = poly;
     const pose = poly.pose ? poly.pose : NO_POSE;
     const earcutPoints: Vec3[] = getEarcutPoints(points);
     const polyPoints: Point[] = earcutPoints.map(([x, y, z]) => ({ x, y, z }));
-    const color = poly.color;
     triangles.push({
+      id,
       points: polyPoints,
       pose,
       color: { r: color[0], g: color[1], b: color[2], a: color[3] },
       scale: DEFAULT_SCALE,
     });
   }
-
   return <Triangles getHitmapId={getHitmapId}>{triangles}</Triangles>;
```

Comment:
here we should spread the rest of the react props in order to pass through the mouse handlers.